### ISSUE TITLE
Make a Simple Html page for the CLI tool demonstrating its usage

### ISSUE THREAD

**Issue by AmitPress** [AmitPress/randomsamplepicker#5]

**Comment by ToobaJamal** [comment]
Could you please assign this to me?

**Comment by AmitPress** [comment]
> Could you please assign this to me?

Yeah of course...  DONE... Feel free to ask me any further query.

**Comment by HamnaJabbar** [comment]
Hi , @AmitPress ,Is it supposed to make a page in HTML to create awareness and to tell about about the CLI tool?

**Comment by AmitPress** [comment]
> Hi , @AmitPress ,Is it supposed to make a page in HTML to create awareness and to tell about about the CLI tool?

Yes you get it right  😸

**Comment by HamnaJabbar** [comment]
Can i contribute to it?

**Comment by AmitPress** [comment]
> Can i contribute to it?

Yeah of course... Multiple contribution against single issue is always accepted... Just follow the standard practices everything will be okay 😃

**Comment by HamnaJabbar** [comment]
@AmitPress ! I've just made a contribution . let me know if you have any feedback or suggestions.Thankyou

**Comment by AmitPress** [comment]
> @AmitPress ! I've just made a contribution . let me know if you have any feedback or suggestions.Thankyou

send pr!

**Comment by HamnaJabbar** [comment]
Done,have you received now?
